docusaurus/docs/how-to-guides/data-source-plugins/add-support-for-externally-shared-dashboards.md

```diff
@@ -0,0 +1,63 @@
+---
+id: add-support-for-externally-shared-dashboards
+title: Add support for externally shared dashboards
+description: How to add support for externally share dashboards (previously called Public dashboards).
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - externally shared dashboards
+  - public dashboards
+  - data source
+  - datasource
+---
+
+Externally shared dashboards (previously called Public dashboards) retrieve the query from the backend, instead of receiving it from the frontend. That is due to security purposes, to not expose sensitive data and also not be able to perform unauthorized queries.
```

Comment:
```suggestion
Externally shared dashboards (previously called Public dashboards) retrieve a query from the backend, instead of receiving it from the frontend. This is to avoid exposing sensitive data and performing unauthorized queries.
```

Reply:
I would leave `**the** query`, as it's the panel query to be performed